### ISSUE TITLE
Focus indicator for Layer's hidden anchor in Firefox

### ISSUE THREAD

**Issue by stephanpelikan** [grommet/grommet#7128]
In Firefox there is a focus indicator visible for the Layer's hidden anchor. It is not visible in Chrome. I didn't test in other Browsers.

I could manage to remove it using this code:

```ts
const NoFocusIndicatorLayer = styled(Layer)`
  a:focus {
    outline: none;
  }
`;
```

But maybe you like to add this to the [HiddenAnchor](https://github.com/grommet/grommet/blob/master/src/js/components/Layer/LayerContainer.js#L26)?

### Expected Behavior

No focus indicator for focusable but hidden elements added by Grommet.

### Actual Behavior

A focus indicator is visible.

### URL, screen shot, or Codepen exhibiting the issue

<img width="224" alt="Bildschirmfoto 2024-02-12 um 06 52 34" src="https://github.com/grommet/grommet/assets/5190548/439cfc54-c310-4961-824c-efeaa5f03d94">

(see blue dot where the red arrow is pointing to)

### Steps to Reproduce

Use this component:

```ts
const LoadingIndicator = () => {
  return (
      <NoFocusIndicatorLayer
          plain
          animate={false}
          responsive={false}
          modal={true}>
        <Box
            round='medium'
            background={ { color: 'rgba(0, 0, 0, 0.3)' } }>
          <Box
              animation="rotateRight"
              pad='medium'>
            <Cycle
                style={ { marginTop: '3px' } }
                color="white"
                size="large" />
          </Box>
        </Box>
      </NoFocusIndicatorLayer>);
}
```

### Your Environment

- Grommet version: 2.31.0
- Browser Name and version: Firefox Mac 122
- Operating System and version (desktop or mobile): MacOs 14.1.2

**Comment by stephanpelikan** [comment]
Thank you for fixing 👍
